src/validations/html/htmlInspector.js

```diff
@@ -52,10 +82,14 @@ class HtmlInspectorValidator extends Validator {
     }
 
     const {HTMLInspector} = await importLinters();
+
+    HTMLInspector.rules.add('validate-list-children',
+      invalidListChildrenValidator);
```

Comment:
Similarly, project style for multi-line function calls:

```js
HTMLInspector.rules.add(
  'validate-list-children',
  invalidListChildrenValidator,
);
```

Reply:
Done